src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,15 @@
+.product-rate__stars {
```

Comment:
reuse stars block from a previous task. I should have an opportunity to set amount of star from 1 to 5